src/main/java/de/bonndan/nivio/output/threeD/ThreeDRenderController.java

```diff
@@ -29,11 +29,11 @@
     @Autowired
     public ThreeDRenderController(LandscapeRepository landscapeRepository) throws IOException {
         this.landscapeRepository = landscapeRepository;
-        content = new String(Files.readAllBytes(new File("src/main/resources/static/html/3d.html").toPath()));
+        content = IOUtils.toString(getClass().getResourceAsStream("/static/html/3d.html"), StandardCharsets.UTF_8);
     }
 
     @RequestMapping(method = RequestMethod.GET, path = "/render/{landscape}/threeD.html")
-    public ResponseEntity<String> three3Resource(@PathVariable(name = "landscape") final String landscapeIdentifier) throws IOException {
+    public ResponseEntity<String> three3Resource(@PathVariable(name = "landscape") final String landscapeIdentifier) {
```

Comment:
That exception was not thrown according to IntelliJ - am I overlooking something?